core/storage/beam_job/gae_models.py

```diff
@@ -80,33 +80,63 @@
     'UNKNOWN')
 
 
+def _get_new_model_id(model_class):
+    """Generates an ID for a new model.
+
+    Returns:
+        str. The new ID.
+    """
+    for _ in python_utils.RANGE(_MAX_ID_GENERATION_ATTEMPTS):
+        new_uuid = uuid.uuid4().bytes
+        new_id = base64.urlsafe_b64encode(new_uuid).rstrip(b'=').decode('ascii')
```

Comment:
Can you make these just have [0-9A-Za-z] (no - or _) -- maybe just replace them with 'a' or something? Minor thing but we're trying to remove the extra special chars from our IDs in general to make the charset more standardized.

Reply:
Base64 encoding does ask for 2 custom characters, but your suggestion would just leave 62 -- does this mean no other kind of character is OK?

Reply:
I think it's valuable to have 64-even characters, base64 is a very convenient way to create IDs and it'd be a shame to lose access to it due to two missing characters.

Reply:
Just use 'ab' as the two custom characters.

Reply:
See e.g. utils.convert_to_hash()

Reply:
Done.